metadata.json

```diff
@@ -42,10 +42,9 @@
       ]
     },
     {
-      "operatingsystem": "Scientific",
+      "operatingsystem": "SLES",
```

Comment:
Don't you have 2 blocks of SLES now?

Reply:
Apologies - a copy / paste in my script 😭  🤦 

Fix incoming...